decidim-core/spec/cells/decidim/diff_cell_spec.rb

```diff
@@ -61,72 +61,64 @@
     end
 
     context "with diff_view_unified_unescaped" do
-      let(:html) { subject.find("#diff-for-body") }
-
-      it "renders potentially safe HTML tags unescaped" do
-        expect(html).to have_selector("em", text: "em")
-        expect(html).to have_selector("u", text: "u")
-        expect(html).to have_selector("strong", text: "strong")
+      it "renders potentially safe subject tags unescaped" do
+        within "#diff-for-body" do
+          expect(subject).to have_selector("em", text: "em")
+          expect(subject).to have_selector("u", text: "u")
+          expect(subject).to have_selector("strong", text: "strong")
+        end
       end
 
       it "sanitizes potentially malicious HTML tags" do
-        expect(html).not_to have_selector("script", visible: :all)
-        expect(html).to have_content("alert('SCRIPT')")
+        within "#diff-for-body" do
+          expect(subject).not_to have_selector("script", visible: :all)
+          expect(subject).to have_content("alert('SCRIPT')")
+        end
       end
     end
 
     context "with diff_view_unified_escaped" do
-      let(:diff_html_value) { "escaped" }
-      let(:html) { subject.find("#diff-for-body") }
-
       it "sanitizes potentially safe HTML tags" do
-        expect(html).not_to have_selector("em")
-        expect(html).to have_content("em")
-        expect(html).not_to have_selector("u")
-        expect(html).to have_content("u")
-        expect(html).not_to have_selector("strong")
-        expect(html).to have_content("strong")
+        expect(subject).to have_selector("em", text: "em")
+        expect(subject).to have_selector("u", text: "u")
+        expect(subject).to have_selector("strong", text: "strong")
       end
 
       it "sanitizes potentially malicious HTML tags" do
-        expect(html).not_to have_selector("script", visible: :all)
-        expect(html).to have_content("alert('SCRIPT')")
+        within "#diff-for-body" do
+          expect(subject).not_to have_selector("script", visible: :all)
+          expect(subject).to have_content("alert('SCRIPT')")
+        end
       end
     end
 
     context "with diff_view_split_unescaped" do
-      let(:diff_mode_value) { "split" }
-      let(:html) { subject.find("#diff-for-body") }
-
       it "renders potentially safe HTML tags unescaped" do
         expect(html).to have_selector("em", text: "em")
```

Comment:
The error message from the CI is 

>        undefined local variable or method `html' for #<RSpec::ExampleGroups::DecidimDiffCell::ViewUnescapedHtml::WithDiffViewSplitEscaped "sanitizes potentially safe HTML tags" (./spec/cells/decidim/diff_cell_spec.rb:111)> 

That's because we're removing the `html` variable, you need to use `subject`.

Apart from that you need to add the `within "#diff-for-body" do` for this block so it's the same code